### PR TITLE
fix: fix wrong cli metadata

### PR DESCRIPTION
**Pull request by amrbashir**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: tooling/cli/metadata.json [hunk 1]
```diff
@@ -4,5 +4,5 @@
     "node": ">= 10.0.0"
   },
   "tauri": "1.1.0",
-  "tauri-build": "1.1.4"
+  "tauri-build": "1.1.1"
```
Comment (lucasfernog) [inline tooling/cli/metadata.json]: current tauri-build is 1.1.0

Comment (lucasfernog) [inline tooling/cli/metadata.json]:
```suggestion
  "tauri-build": "1.1.0"
```

Comment (amrbashir) [reply tooling/cli/metadata.json]: fix a mistake by doing another mistake, godd job me lol

Comment (FabianLars) [reply tooling/cli/metadata.json]: relatable

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**